pkg/config/config.go

```diff
@@ -206,7 +211,7 @@ type IngressConfig struct {
 	RTMPBaseURL string `yaml:"rtmp_base_url"`
 }
 
-func NewConfig(confString string, strictMode bool, c *cli.Context) (*Config, error) {
+func NewConfig(c *cli.Context, baseFlags []cli.Flag, strictMode bool) (*Config, error) {
```

Comment:
Does this work okay (i. e. passing in `cli.Context` and reading `config-body`) if this module is used from `cloud`?

Reply:
good point. I've reverted this change.